inductiva/resources/README.md

```diff
@@ -58,17 +71,17 @@ price_per_hour = mg.estimate_cloud_cost()
 # Start the machines
 mg.start()
 
-# Create a list of e.g. 100 protein files you want to simulate
-my_pdb_files = get_my_pdb_file_paths()
+# Download the insulin protein (ID - "1ZNI") from RCSB database
+insulin_pdb_file = inductiva.molecules.utils.download_pdb_from_rcsb(pdb_id="1ZNI")
 tasks = []
 
 # And simulate the solvation of the proteins on your newly started machines
 for my_pdb_file in my_pdb_files:
```

Comment:
This does not work, since you do not have any `my_pdb_files`. To examplify do just with range and change the temperature for example.

Reply:
Good idea! Thanks!